RibonDAO/app
# New profile

# Description

- Full implementation of the new profile

## Did you use styled-components?


## Type of change

Please delete options that are not relevant.


## How to test

- Go to the impact page
- Check the new statistics: tickets donated and days doing good
- Check the ticket and config button on top
- Check the impact list by projects and inactive projects

## Demo video

https://github.com/RibonDAO/app/assets/57116106/921e9ba6-110e-4f15-ad3a-8ef596879cba
